content/en/docs/refguide/modeling/pages/page/_index.md

```diff
@@ -190,6 +190,28 @@ It has the following features which are not available in **Structure mode**:
 
     {{< figure src="/attachments/refguide/modeling/pages/page/conditional-visibility.jpg" alt="Show conditional visibility" >}}
 
+* **X-ray mode** to visualize the structure of a page
+
+#### 3.2.1 X-ray mode (beta)
+
+**X-ray mode** is a way to visualize certain structures of a page while in **Design mode**. When turned on, certain widgets appear bigger and have an extra outline, making them easier to see, select and work with when placing new widgets within them. **X-ray mode** affects structures such as **Container**, **Layout Grid** and **Data View** widgets. These extra effects are removed when **X-ray mode** is turned off.
+
+**X-ray mode** can be turned on or off by using the button in the top bar. It can also be turned on or off using the keyboard shortcut `ctrl + alt + x` (on Windows) or `cmd + alt + x` (on Mac.)
+
+{{% alert color="info" %}}
+**X-ray mode** is currently in beta. The feature is enabled by default on Mac and disabled by default on Windows. However, it can be enabled or disabled in the `Preferences...` menu, in the `New features` section.
```

Comment:
I would put this alert before telling users how to use the feature, that way it flows as follows
Introduction
How to enable/disable the feature
How to use the feature (with examples)